lib/billy/capybara/capybara.rb

```diff
@@ -0,0 +1,52 @@
+require 'billy'
+
+module Billy
+  def self.register_drivers_capybara
```

Comment:
I'd keep this as `register_drivers` because then the code doesn't change when registering Capybara or registering Watir.